server/src/main/java/org/elasticsearch/indices/breaker/HierarchyCircuitBreakerService.java

```diff
@@ -585,19 +605,27 @@ static long fallbackRegionSize(JvmInfo jvmInfo) {
             return regionSize;
         }
 
+        @SuppressForbidden(reason = "Prefer full GC to OOM or CBE")
+        private static void performFullGC() {
```

Comment:
Should this be injectable? It would make unit testing easier; however I don't like too much the idea of injecting just a method - maybe we should separate `OverLimitStrategy`es and had a "composite" class over them.

Reply:
let's leave it as it, for now